examples/Carbon_Flux.ipynb

```diff
@@ -81,65 +223,411 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "dailies = load_fluxnet_site(sites[0])\n",
-    "print('Example FLUXNET data for site {site}:'.format(site=sites[0]))\n",
-    "dailies.head()"
+    "futures = client.map(load_fluxnet_site, sites, one=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "succeeded = [f for f in futures if not f.exception()]\n",
+    "failed = [f for f in futures if f.exception()]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dfs = client.gather(succeeded)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "## Merge data\n",
     "\n",
-    "Many of the FluxNet variables are expressed relative to a historical reference called ERA: \n",
-    "\n",
-    "* ERA-15: December 1978 to February 1994.\n",
-    "* ERA-40: Reanalysis from September 1957 through August 2002\n",
-    "  \n",
-    "Relevant links: [ERA](https://en.wikipedia.org/wiki/ECMWF_re-analysis) and [NCAR_Reanalysis](https://en.wikipedia.org/wiki/NCEP/NCAR_Reanalysis).\n",
-    "\n",
-    " \n",
-    "The goal in the matlab code is to build a super-array ``allData`` that spans 8766 Days x 22 Variables and 209 sites. Note that the 8766 span is derived from the *FluxNet* data not ERA. In the repository, I see 210 files with 21 variables. Some of the key variables:\n",
+    "Once the data are loaded in, they need to be joined with the metadata relating to each site."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df = pd.concat(dfs)\n",
+    "\n",
+    "# create a little dataframe for mapping categorical variables\n",
+    "a = np.zeros((len(igbp_codes), len(igbp_codes)), int)\n",
+    "np.fill_diagonal(a, 1)\n",
+    "categorical_igbp_mapper = pd.DataFrame(index=igbp_codes, columns=igbp_codes, data=a)\n",
+    "categorical_igbp_mapper.rename_axis('igbp', inplace=True)\n",
+    "\n",
+    "# add metadata to the big dataframe\n",
+    "df = pd.merge(df, metadata, on='site')\n",
+    "df = pd.merge(df, categorical_igbp_mapper, on='igbp')\n",
+    "\n",
+    "print(\"df shape =\", df.values.shape)\n",
+    "if float('nan') in df.columns:\n",
+    "    print('nan in df.columns, removing')\n",
+    "    df.pop(float('nan'))"
+   ]
```

Comment:
What does a column `nan` mean?

Reply:
@stsievert I am taking this part out unless you know why it is there. I agree that it doesn't really make sense.